### PR TITLE
added tasrie it services as support partner

### PR DESCRIPTION
**Pull request by tasrieit**
* [ ] I added CHANGELOG entry for this change.
* [x] Change is not relevant to the end user.

### CODE REVIEW

**Comment by MichaHoffmann** [comment]
Just as a note: you can just 'git commit --amend -s' and force push to sign the commits with the PR still open, no need to reopen the PRs

**Review by MichaHoffmann — APPROVED** [review]
Lgtm

**Review by saswatamcode — APPROVED** [review]
Thanks!

File: website/layouts/support/list.html [hunk 1]
```diff
@@ -13,6 +13,10 @@ <h2><a href="{{ .Permalink }}">Support and Training</a></h2>
                          <a href="https://o11y.eu">
                              <img src="{{ "/o11y.svg" | relURL }}"  alt="" style="width: auto; height: auto; border-color: #ccc;">
                          </a>
+                         <a href="https://tasrieit.com">
+                             <img src="{{ "/tasrie-it-services.png" | relURL }}"  alt="" style="width: auto; height: auto; border-color: #ccc;">
+                         </a>
+                         tasrie-it-services.png
```
Comment (saswatamcode) [inline website/layouts/support/list.html:19]: I think this line is extraneous, shows up on the site as so. Could you kindly resize the image and remove the test @tasrieit?
Screenshot 2024-01-01 at 6 32 05 PM

Comment (tasrieit) [reply website/layouts/support/list.html:19]: i will fix it thank you

Comment (saswatamcode) [inline website/layouts/support/list.html:19]:
```suggestion
```

**Review by saswatamcode — APPROVED** [review]

**Pull request merged**